x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/ltr/LearningToRankRescorerBuilder.java

```diff
@@ -249,6 +250,11 @@ protected LearningToRankRescorerContext innerBuildContext(int windowSize, Search
         return new LearningToRankRescorerContext(windowSize, LearningToRankRescorer.INSTANCE, learningToRankConfig, localModel, context);
     }
 
+    @Override
```

Comment:
I think we could omit this?

Reply:
Do you mean, omitting the `@Override`?

Reply:
Yes.

Reply:
Yes that's what I meant - but it looks like my comment may be out of date 😄